### PR TITLE
[MKT-474]: feat/sustainability Landing Page

### PR DESCRIPTION
**Pull request by jaaaaavier**
Created Sustainability Landing Page

- New green-cloud-computing page
- New components for the green-cloud-computing page
- Added links to the navbar and footer

### CODE REVIEW

File: src/components/layout/navbars/Navbar.tsx [hunk 1]
```diff
@@ -353,6 +353,11 @@ export default function Navbar(props: Readonly<NavbarProps>) {
                                     <p>{props.textContent.ourValues.openSource}</p>
                                   </div>
                                 </Link>
+                                <Link href="/green-cloud-computing" locale={props.lang} passHref legacyBehavior>
+                                  <div className="flex flex-row space-x-2">
```
Comment (xabg2) [inline src/components/layout/navbars/Navbar.tsx:357]: Why is this text in a div?

File: src/components/sustainability/FeatureSection.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
+import { ArrowsClockwise, CloudArrowUp, Drop, Gift, Lightning } from '@phosphor-icons/react';
+import RevealX from '../components/RevealX';
+import Image from 'next/image';
+import { getImage } from '@/lib/getImage';
+import { SustainabilityText } from '@/assets/types/sustainabilty';
+
+export interface FeatureSection {
```
Comment (xabg2) [inline src/components/sustainability/FeatureSection.tsx]:
```
interface FeatureSectionProps{
  textContent: SustainabilityText['FeatureSectionV2'];
}

```

then, 
`const FeatureSection = ({ textContent }: FeatureSectionProps) => {`

File: src/components/sustainability/FeatureSection.tsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  return (
+    <section className="overflow-hidden px-5">
+      <div className="flex flex-col items-center justify-center space-y-16 py-20">
+        <div className="flex max-w-[850px] flex-col items-center justify-center space-y-6 text-center">
+          <p className="text-5xl font-semibold text-gray-100">{textContent.title}</p>
+          <p className="text-xl text-gray-80">{textContent.description}</p>
+        </div>
+        <div>
+          {cards.map((card, index) => {
+            const Icon = card.icon;
```
Comment (xabg2) [inline src/components/sustainability/FeatureSection.tsx]: Why? You can use `card.icon` directly.

File: src/components/sustainability/FeatureSectionV2.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
+import { SustainabilityText } from '@/assets/types/sustainabilty';
+
+export interface FeatureSectionV2 {
```
Comment (xabg2) [inline src/components/sustainability/FeatureSectionV2.tsx]: Use the interface. Also, use Props at the end (`FeatureSectionV2Props`)

File: src/components/sustainability/HeroSection.tsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import { SustainabilityText } from '@/assets/types/sustainabilty';
+import Header from '@/components/shared/Header';
+
+export interface HeroSection {
```
Comment (xabg2) [inline src/components/sustainability/HeroSection.tsx]: Use the interface (`HeroSectionProps`)

**Review by xabg2 — CHANGES_REQUESTED** [review]

File: src/components/layout/navbars/Navbar.tsx (hunk 1 above)
Comment (jaaaaavier) [reply src/components/layout/navbars/Navbar.tsx:357]: The other elements are like this, to respect the structure.

File: src/components/sustainability/FeatureSection.tsx (hunk 2 above)
Comment (jaaaaavier) [reply src/components/sustainability/FeatureSection.tsx]: Done

File: src/components/sustainability/FeatureSection.tsx (hunk 3 above)
Comment (jaaaaavier) [reply src/components/sustainability/FeatureSection.tsx]: Done

File: src/components/sustainability/HeroSection.tsx (hunk 5 above)
Comment (jaaaaavier) [reply src/components/sustainability/HeroSection.tsx]: Done

File: src/components/sustainability/FeatureSectionV2.tsx (hunk 4 above)
Comment (jaaaaavier) [reply src/components/sustainability/FeatureSectionV2.tsx]: Done

**Pull request closed without merging**